frontend/src/components/Search.tsx

```diff
@@ -85,6 +85,10 @@ const Search: FunctionComponent = () => {
       size="sm"
       data={results}
       value={query}
+      limit={30}
+      // uncomment following line after upgrade to Mantine 7.x or higher
```

Comment:
Can you prefix this with TODO:

```suggestion
      // TODO: uncomment following line after upgrade to Mantine 7.x or higher
```